syntax = "proto3";

package processor;

message Point {
    repeated int32 pixelPoint = 1;
}

service ImageProcessor {
    rpc GetAveragePixel(Point) returns (AveragePixel) {}
}

message AveragePixel {
    int32 average = 1;
}